common/tools/dev-tool/src/templates/sampleReadme.md.ts

```diff
@@ -134,9 +136,8 @@ function exampleNodeInvocation(info: SampleReadmeConfiguration) {
     .map((envVar) => `${envVar}="<${envVar.replace(/_/g, " ").toLowerCase()}>"`)
     .join(" ");
 
-  return `${envVars} node ${
-    info.useTypeScript ? "dist/" : ""
-  }${firstModule.relativeSourcePath.replace(/\.ts$/, ".js")}`;
+  return `${envVars} node ${info.useTypeScript ? "dist/" : ""
+    }${firstModule.relativeSourcePath.replace(/\.ts$/, ".js")}`;
```

Comment:
are those changed by eslint or prettier ?

Reply:
maybe it's changed by Prettier